app/services/group/v1x1_seed.rb

```diff
@@ -0,0 +1,74 @@
+module Group
+  class V1x1Seed
+    attr_reader :status
+    CATALOG_ADMINISTRATOR_GROUP = "Catalog Administrators".freeze
+
+    def initialize
+      @request = Insights::API::Common::Request.current
+      raise Catalog::NotAuthorized unless @request.user.org_admin?
+    end
+
+    def tenant
+      @request.tenant
+    end
+
+    def code(status)
+      @status = status
+    end
+
+    def process
+      lookup_groups
+      mark_as_seeded
+
+      if @seeded.data.present?
+        code(204)
+      else
+        run_seeding
+        add_user_to_catalog_admin_group
+      end
+
+      self
+    end
+
+    private
+
+    def run_seeding
+      seeded = Insights::API::Common::RBAC::Seed.new(Rails.root.join('data', 'rbac_catalog_seed.yml')).process
+      if seeded
+        RbacSeed.create!(:external_tenant => @request.tenant)
```

Comment:
Should `RbacSeed` use `acts_as_tenant(:tenant)?`  Or if we allow this to run more than once can we just drop the class altogether?